app/jobs/cache/feed_job.rb

```diff
@@ -1,19 +1,25 @@
+# frozen_string_literal: true
 module Cache
   class FeedJob < ApplicationJob
+    def self.custom_feeds
+      {
+        custom: ::Feed.all.each_with_object({}) do |feed, hash|
+          hash[feed.slug.to_sym] = feed.url
+        end
+      }
+    end
+
     URLS = {
       blog: {
         all: 'http://blog.europeana.eu/feed/',
-        art: 'http://blog.europeana.eu/tag/art-history/feed/',
+        art: 'http://blog.europeana.eu/tag/art/feed/',
         fashion: 'http://blog.europeana.eu/tag/fashion/feed/',
         music: 'http://blog.europeana.eu/tag/music/feed/'
       },
       exhibitions: %i(de en).each_with_object({}) do |locale, hash|
         hash[locale] = (ENV['EXHIBITIONS_HOST'] || 'http://www.europeana.eu') + "/portal/#{locale}/exhibitions/feed.xml"
-      end,
-      tumblr: {
-        fashion: 'http://europeanafashion.tumblr.com/rss'
-      }
-    }
+      end
+    }.merge custom_feeds
```

Comment:
This will not work as intended. Once `Cache::FeedJob::URLS` has been set, it will not be altered to reflect any new/modified/deleted feed URLs.

I suggest removing the `.custom_feeds` method and instead just calling `Feed.all` where you have called `Cache::FeedJob::URLS[:custom]`.